src/python/WMCore/MicroService/MSPileup/MSPileupTaskManager.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python
+"""
+File       : MSPileupTaskManager.py
+Author     : Valentin Kuznetsov <vkuznet AT gmail dot com>
+Description: MSPileupTaskManager handles MSPileupTasks
+
+In particular, it perform the following tasks each polling cycle:
+    - fetches pileup sizes for all pileup documents in database back-end
+    - update RSE quotas
+    - perform monitoring task
+    - perform task for active pileups using up-to-date RSE quotas
+    - perform task for inactive pileups
+"""
+
+# system modules
+import os
+from threading import current_thread
+
+# WMCore modules
+from WMCore.MicroService.MSCore.MSCore import MSCore
+from WMCore.MicroService.DataStructs.DefaultStructs import PILEUP_REPORT
+from WMCore.MicroService.MSPileup.MSPileupData import MSPileupData
+from WMCore.MicroService.MSPileup.MSPileupTasks import MSPileupTasks
+from WMCore.MicroService.MSTransferor.DataStructs.RSEQuotas import RSEQuotas
+from WMCore.MicroService.Tools.PycurlRucio import getPileupContainerSizesRucio, getRucioToken
+from WMCore.Services.Rucio.Rucio import Rucio
+
+
+class MSPileupTaskManager(MSCore):
+    """
+    MSPileupTaskManager handles MSPileup tasks
+    """
+
+    def __init__(self, msConfig, **kwargs):
+        super().__init__(msConfig, **kwargs)
+        self.marginSpace = msConfig.get('marginSpace', 1024**4)
+        self.rucioAccount = msConfig.get('rucioAccount', 'ms-pileup')
```

Comment:
this account should change to `wmcore_transferor`. 
I do agree though that we should adopt a specific account at some point, but that will require further changes including WMAgent and Global Workqueue.

Reply:
done